RELEASE_NOTES.md

```diff
@@ -1,8 +1,8 @@
-#### 1.4.0 TBD ####
+#### 1.3.1 TBA ####
```

Comment:
Change TBA to the date of your update, or today. In the format as used further down in the file

Reply:
Also add a note about supporting the 1.3.5 version of Akka